src/bacnet/datalink/bvlc.c

```diff
@@ -2626,20 +2626,23 @@ const char *bvlc_result_code_name(uint16_t result_code)
  * @brief Encode a BBMD Address for Network Port object
  * @param apdu - the APDU buffer
  * @param apdu_size - the APDU buffer size
- * @param ip_address - IP address and port number
+ * @param bbmd_address - HostNPort FD BBMD Address
  * @return length of the APDU buffer
  */
 int bvlc_foreign_device_bbmd_host_address_encode(
-    uint8_t *apdu, uint16_t apdu_size, const BACNET_IP_ADDRESS *ip_address)
+    uint8_t *apdu, uint16_t apdu_size, const BACNET_HOST_N_PORT * const bbmd_address)
 {
+#if (__STDC__) && (__STDC_VERSION__ >= 199901L)
```

Comment:
Would __STDC_VERSION__ be defined if __STDC__ was not?

Reply:
From what I saw the two come together when the standard is set to lets say c90.